zng/docs/spec.md

```diff
@@ -74,8 +75,7 @@ ZNG is more expressive than JSON in that any JSON input
 can be mapped onto ZNG and recovered by decoding
 that ZNG back into JSON, but the converse is not true.
 
-The ZNG design [was motivated by](./zeek-compat.md)
-and is compatible with the
+The ZNG design was motivated by and [is compatible with](./zeek-compat.md) the
```

Comment:
I'm just attaching the link to better words here: The "Zeek compatibility" doc is more about compatibility than motivation.